grid2op/tests/issue_140.py

```diff
@@ -0,0 +1,141 @@
+import grid2op
+import numpy as np
+#!/usr/bin/env python3
+
+import grid2op
+import unittest
+import numpy as np
+import warnings
+import pdb
+
+
+class Issue140Tester(unittest.TestCase):
+    def test_issue_140(self):
+
+        # TODO change that
+        envs = grid2op.list_available_local_env()
+        env_name = "l2rpn_neurips_2020_track1_small"
+        if env_name not in envs:
+            # the environment is not downloaded, I skip this test
+            self.skipTest("{} is not downloaded".format(env_name))
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore")
+            env = grid2op.make(env_name)
+        ts_per_chronics = 2016
+
+        seed = 725
+        np.random.seed(seed)
+        env.seed(seed)
+
+        env.set_id(np.random.randint(100000))
+        _ = env.reset()
+
+        max_day = (env.chronics_handler.max_timestep() - 2016) // 288
+        start_timestep = np.random.randint(max_day) * 288 - 1  # start at 00:00
+        if start_timestep > 0:
+            env.fast_forward_chronics(start_timestep)
+
+        obs = env.get_obs()
+
+        done = False
+        steps = 0
+        do_nothing = env.action_space({})
+
+        for i in range(119):
+            obs, reward, done, info = env.step(do_nothing)
+            assert not done
+
+        act0 = env.action_space.from_vect(np.array(
```

Comment:
If possible, print these, and rewrite the action declaration using human readable dict format

Reply:
A bit overkill but why not